test/smoke/process-usage/src/test/fixtures.js

```diff
@@ -0,0 +1,229 @@
+'use strict';
+
+const timewindow = require('abacus-timewindow');
+
+const { times } = require('underscore');
+
+const { fixturesCfg } = require('./env-config');
+
+// The scaling factor of each time window for creating the date string
+// [Second, Minute, Hour, Day, Month]
+const _slack = () =>
+  /^[0-9]+[MDhms]$/.test(fixturesCfg.slack)
+    ? {
+      scale: fixturesCfg.slack.charAt(fixturesCfg.slack.length - 1),
+      width: fixturesCfg.slack.match(/[0-9]+/)[0]
+    }
+    : {
+      scale: timewindow.dimension.min,
+      width: 10
+    };  
+
+const _initWindows = (win, dimension) => {
+  const windows = [win];
+
+  if(fixturesCfg.windowsSizes && fixturesCfg.windowsSizes[dimension])
```

Comment:
I would extract `fixturesCfg.windowsSizes[dimension]` into a separate temporary constant.

Reply:
After a discussion we decided to stay with this implementation. Because it causes double check: 

```
const dimension = fixturesCfg.windowsSize ?  fixturesCfg.windowsSizes[dimension] : undefined;

if(dimension) 
...
```